src/modm/platform/core/nrf/startup_platform.c.in

```diff
@@ -0,0 +1,153 @@
+//        for your device located at https://infocenter.nordicsemi.com/index.jsp  */
+//     if (nrf52_errata_98()){
+//         *(volatile uint32_t *)0x4000568Cul = 0x00038148ul;
+//     }
+
+//     /* Workaround for Errata 103 "CCM: Wrong reset value of CCM MAXPACKETSIZE" found at the Errata document
+//        for your device located at https://infocenter.nordicsemi.com/index.jsp  */
+//     if (nrf52_errata_103()){
+//         NRF_CCM->MAXPACKETSIZE = 0xFBul;
+//     }
+
+//     /* Workaround for Errata 115 "RAM: RAM content cannot be trusted upon waking up from System ON Idle or System OFF mode" found at the Errata document
+//        for your device located at https://infocenter.nordicsemi.com/index.jsp  */
+//     if (nrf52_errata_115()){
+//         *(volatile uint32_t *)0x40000EE4ul = (*(volatile uint32_t *)0x40000EE4ul & 0xFFFFFFF0ul) | (*(uint32_t *)0x10000258ul & 0x0000000Ful);
+//     }
+
+//     /* Workaround for Errata 120 "QSPI: Data read or written is corrupted" found at the Errata document
+//        for your device located at https://infocenter.nordicsemi.com/index.jsp  */
+//     if (nrf52_errata_120()){
+//         *(volatile uint32_t *)0x40029640ul = 0x200ul;
+//     }
+
+//     /* Workaround for Errata 136 "System: Bits in RESETREAS are set when they should not be" found at the Errata document
+//        for your device located at https://infocenter.nordicsemi.com/index.jsp  */
+//     if (nrf52_errata_136()){
+//         if (NRF_POWER->RESETREAS & POWER_RESETREAS_RESETPIN_Msk){
+//             NRF_POWER->RESETREAS =  ~POWER_RESETREAS_RESETPIN_Msk;
+//         }
+//     }
+
+    /* Enable the FPU if the compiler used floating point unit instructions. __FPU_USED is a MACRO defined by the
+     * compiler. Since the FPU consumes energy, remember to disable FPU use in the compiler if floating point unit
+     * operations are not used in your code. */
+    #if (__FPU_USED == 1)
+        SCB->CPACR |= (3UL << 20) | (3UL << 22);
+        __DSB();
+        __ISB();
+    #endif
+
+    /* Configure NFCT pins as GPIOs if NFCT is not to be used in your code. If CONFIG_NFCT_PINS_AS_GPIOS is not defined,
+       two GPIOs (see Product Specification to see which ones) will be reserved for NFC and will not be available as
+       normal GPIOs. */
+    #if defined (CONFIG_NFCT_PINS_AS_GPIOS)
+        if ((NRF_UICR->NFCPINS & UICR_NFCPINS_PROTECT_Msk) == (UICR_NFCPINS_PROTECT_NFC << UICR_NFCPINS_PROTECT_Pos)){
+            NRF_NVMC->CONFIG = NVMC_CONFIG_WEN_Wen << NVMC_CONFIG_WEN_Pos;
+            while (NRF_NVMC->READY == NVMC_READY_READY_Busy){}
+            NRF_UICR->NFCPINS &= ~UICR_NFCPINS_PROTECT_Msk;
+            while (NRF_NVMC->READY == NVMC_READY_READY_Busy){}
+            NRF_NVMC->CONFIG = NVMC_CONFIG_WEN_Ren << NVMC_CONFIG_WEN_Pos;
+            while (NRF_NVMC->READY == NVMC_READY_READY_Busy){}
+            NVIC_SystemReset();
+        }
+    #endif
```

Comment:
Should probably be moved out of startup and into a `modm::platform` function or class. Seems a bit harsh with the SystemReset during system boot.